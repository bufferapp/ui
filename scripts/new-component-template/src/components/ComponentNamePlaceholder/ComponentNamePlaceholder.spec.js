/* eslint-disable react/jsx-filename-extension */
import snap from 'jest-auto-snapshots'
import 'jest-styled-components'

import ComponentNamePlaceholder from './ComponentNamePlaceholder'

snap(ComponentNamePlaceholder, './ComponentNamePlaceholder.jsx')
