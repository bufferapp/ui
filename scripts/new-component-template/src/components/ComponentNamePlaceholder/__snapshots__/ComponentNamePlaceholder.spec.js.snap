// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`jest-auto-snapshots > ComponentNamePlaceholder Matches snapshot when passed only required props 1`] = `
<ForwardRef(styled.div)>
  <NodeFixture />
</ForwardRef(styled.div)>
`;
